Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -102,95 +104,78 @@ public async Task<IEnumerable<ProgramInfo>> GetProgramsAsync(ListingsProviderInf
             var requestString = _jsonSerializer.SerializeToString(requestList);
             _logger.LogDebug("Request string for schedules is: {RequestString}", requestString);
 
-            var httpOptions = new HttpRequestOptions()
-            {
-                Url = ApiUrl + "/schedules",
-                UserAgent = UserAgent,
-                CancellationToken = cancellationToken,
-                LogErrorResponseBody = true,
-                RequestContent = requestString
-            };
-
-            httpOptions.RequestHeaders["token"] = token;
+            using var options = new HttpRequestMessage(HttpMethod.Post, ApiUrl + "/schedules");
+            options.Content = new StringContent(requestString, Encoding.UTF8, MediaTypeNames.Application.Json);
+            options.Headers.TryAddWithoutValidation("token", token);
+            using var response = await Send(options, true, info, cancellationToken).ConfigureAwait(false);
+            await using var responseStream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
```

Comment:
Maybe an unrelated change, but have you considered using `ReadAsAsync<List<ScheduleDirect.Day>` instead?

Reply:
ttps://www.nuget.org/packages/Microsoft.AspNet.WebApi.Client/  has Newtonsoft.Json (https://github.com/dotnet/aspnetcore/pull/6552#issuecomment-453177732) so they removed it as of 3.0. I don't think rolling our own extension method would be too difficult, but that can be pushed to another PR